operations/mimir-mixin/alerts/continuous-test.libsonnet

```diff
@@ -0,0 +1,53 @@
+(import 'alerts-utils.libsonnet') {
+  groups+: [
+    {
+      name: 'mimir_continuous_test',
+      rules: [
```

Comment:
WDYT about adding an alert that tracks whether the continuous test is running at all? Something like `absent(sum(rate(mimir_continuous_test_writes_failed_total[5m]))) or absent(...)`

Reply:
There are a couple of reasons why I didn't add it:
- The tool is optional. Having it not deployed is fine.
- `absent()` is tricky in Prometheus. As query as you suggested wouldn't work to detect if it's not running in a specific namespace. We would need to run that query for every namespace, like `absent(metric{namespace="type-an-exact-namespace"}))` and that's a pattern we avoided so far

Reply:
Could we use something like:

```
(sum by (cluster, namespace) (mimir_continuous_test_writes_failed_total offset 30m)) unless sum by (cluster, namespace) (mimir_continuous_test_writes_failed_total)
```

(I tried this on my personal Prometheus and it works)

This would fire for 30m after it's down. Not perfect, but it's something. (We could evolve the query to fire during more time or whatever).

Anyway, it's just an idea.